src/libp11.h

```diff
@@ -378,6 +378,7 @@ extern int PKCS11_generate_random(PKCS11_SLOT *slot, unsigned char *r, unsigned
 /*
  * PKCS#11 implementation for OpenSSL methods
  */
+extern void PKCS11_set_claim_module_token_fips(void);
```

Comment:
All exported symbols should be listed in `libp11.exports`.  In only works with our engine, because the engine is no longer linked dynamically with libp11.

Reply:
Fixed.